test/test_utils.c

```diff
@@ -106,6 +106,12 @@ int mp_msg_find_level(const char *s) {return 0;};
 int mp_msg_level(struct mp_log *log) {return 0;};
 void mp_write_console_ansi(void) {};
 
+void mp_set_avdict(AVDictionary **dict, char **kv)
+{
+    for (int n = 0; kv && kv[n * 2]; n++)
+        av_dict_set(dict, kv[n * 2 + 0], kv[n * 2 + 1], 0);
+}
+
```

Comment:
```suggestion
void mp_set_avdict(AVDictionary **dict, char **kv) {};
```
It's a stub. The tests don't actually use this; it just needs to be defined.